sdk/src/contexts/abstracts/tokenBridge.ts

```diff
@@ -36,51 +73,118 @@ export abstract class TokenBridgeAbstract<TransactionResult> {
   ): Promise<TransactionResult>;
 
   /**
-   * Format an address to a 32-byte array, padded with zeros if needed
-   * @param address The address to format
-   * @returns A 32 byte array of the address, padded with zeros if needed
+   * Format an address to 32-bytes which can be utilized by the Wormhole contracts
+   *
+   * @param address The address as a string
+   * @returns The address as a 32-byte Wormhole address
    */
-  protected abstract formatAddress(address: string): Uint8Array;
+  protected abstract formatAddress(address: string): any;
   /**
-   * Parse an address to the blockchain specific address format
-   * (e.g. 40-byte hex 0xabcd... for evm chains)
-   * @param address The address to parse
-   * @returns The parsed address to the blockchain specific format
+   * Parse an address from a 32-byte Wormhole address which is the standard format for a given chain
+   *
+   * @param address The 32-byte wormhole address
+   * @returns The address in the blockchain specific format
    */
-  protected abstract parseAddress(address: string | Uint8Array): string;
+  protected abstract parseAddress(address: any): string;
 
-  protected abstract formatAssetAddress(address: string): Promise<Uint8Array>;
-  protected abstract parseAssetAddress(address: string): Promise<string>;
+  /**
+   * Format a token address to 32-bytes which can be utilized by the Wormhole contracts
+   *
+   * @param address The token address as a string
+   * @returns The token address as a 32-byte Wormhole address
+   */
+  protected abstract formatAssetAddress(address: string): Promise<any>;
```

Comment:
Perhaps we should clear up why these two methods exists, which is basically that converting some assets to a universal representation might require querying a registry first. I recall getting confused at first when they were introduced for the Sui integration, so clearing it up for other developers might be worth it.